dashboard/lib/services/script_seed.rb

```diff
@@ -198,7 +200,8 @@ def self.seed_from_hash(data)
     # Internal methods and classes below
 
     def self.import_script(script_data)
-      script_to_import = Script.new(script_data.except('seeding_key'))
+      script_to_import = Script.new(script_data.except('seeding_key', 'serialized_at'))
+      script_to_import.seeded_at = script_data['serialized_at']
```

Comment:
nit: `seeded_at` seems misleading to me -- it's not actually the time this script was seeded but instead the timestamp of when this script was last serialized. `last_serialization_timestamp` or something makes more sense to me but I'd be interested to hear other's opinions

Reply:
That's not quite accurate; note that the idea here is that we use this to facilitate the exchange of data between different environments. The script here was actually _never_ serialized because it lives on staging. It's being seeded from a script that was serialized on levelbuilder; `seeded_at` tells us which version of content it was seeded from. Specifically, it tells us when the content that it was originally seeded from was last updated. See [this comment](https://github.com/code-dot-org/code-dot-org/pull/39011/files#diff-22769ab80ff34eaf4aadc1dc87e1b9bdad8b445cec233c9a5a1d2bcc5a7a7978R174-R178) for an explanation

Reply:
maybe `seeded_from` would be more accurate? I'd prefer to stick close to the Rails standard of `*_at`, though

Reply:
Yeah I guess my issue is that `seeded_at` isn't be equal to the time it's actually seeded at. `seeded_from` sounds good to me but up to you.